metron-stellar/stellar-common/README.md

```diff
@@ -54,6 +54,12 @@ The Stellar language supports the following:
 * The ability to have parenthesis to make order of operations explicit
 * User defined functions, including Lambda expressions 
 
+### Boolean Expressions
+
+Similar to python and javascript, empty collections (e.g. `[]` and
```

Comment:
Missing variables vs. NULL variables... this may be confusing.  What this is saying is
we support
- boolean with true or false
- variables that are present but explicitly null or empty
- variables that are NOT present

Reply:
This should be in the first pr

Reply:
Or am i miss understanding the explicit null case?

Reply:
Yeah, what do people think about merging the two PRs into one and reviewing them together?  I split them up anticipating that the community might want one or the other but not necessarily both.  I am thinking that I close #1063 and keep this one.

Thoughts?